test/unittest/tests/wrappers/mock_session.cpp

```diff
@@ -41,6 +43,17 @@ MockSession::buildTestMockSession()
     return std::shared_ptr<MockSession>(new MockSession(sessionImpl, mockConnection));
 }
 
+WT_BLOCK_MGR_SESSION *
+MockSession::setupBlockManagerSession()
+{
+    // Initialize rnd state because block manager requires it.
+    __wt_random_init(&_sessionImpl->rnd);
+    utils::throwIfNonZero(
+      __wt_calloc(nullptr, 1, sizeof(WT_BLOCK_MGR_SESSION), &_sessionImpl->block_manager));
```

Comment:
We should rename `throwIfNonZero` to the current naming convention, however I suggest we do that in a seperate ticket.

Reply:
Hey jeremy, i agree, [WT-13405](https://jira.mongodb.org/browse/WT-13405) will be covering the change.

Reply:
Perfect - thanks Jie.